nncf/experimental/onnx/graph/model_transformer.py

```diff
@@ -61,33 +64,47 @@ def _get_transformation_layout_extra_outputs(
 
         return transformation_layout
 
-    def _apply_transformations(self, transformations: List[TransformationCommand]) -> None:
+    def transform(self, transformation_layout: TransformationLayout) -> onnx.ModelProto:
         """
         Applies transformations by type-callback on the model
 
         :param transformations: lisf of the TransformationCommand transformations
         """
         quantizer_insert_transformations = []
         output_insert_transformations = []
+        bias_correction_transformations = []
+        model_extraction_transformation = None
+
+        transformations = transformation_layout.transformations
 
         for transformation in transformations:
             if isinstance(transformation, ONNXQuantizerInsertionCommand):
                 quantizer_insert_transformations.append(transformation)
             elif isinstance(transformation, ONNXOutputInsertionCommand):
                 output_insert_transformations.append(transformation)
+            elif isinstance(transformation, ONNXBiasCorrectionCommand):
+                bias_correction_transformations.append(transformation)
+            elif isinstance(transformation, ONNXModelExtractionCommand):
+                model_extraction_transformation = transformation
```

Comment:
Is it a valid situation when we have several `ONNXModelExtractionCommand`?

Reply:
In general, we shouldn't have several `ONNXModelExtractionCommand`'s. But it's a good question to discuss.